mlock.go

```diff
@@ -0,0 +1,23 @@
+// +build !windows
+
+package main
+
+import (
+	"fmt"
+	"syscall"
+
+	"github.com/urfave/cli"
+	"golang.org/x/sys/unix"
+)
+
+func enableMlock(mlockBool bool) error {
+
+	if mlockBool {
+		fmt.Printf("mlock bool is: %t \n", mlockBool)
```

Comment:
In testing does this print to stdout while normally using the command?  If so this may not be necessary unless a "debug" flag is provided.

Reply:
it only prints out "mlock bool is: true"  if you pass the -m flag when running the command.  I agree that it can be removed and will remove it.